src/server/plugins/engine/components/TextField.ts

```diff
@@ -51,8 +54,15 @@ export class TextField extends FormComponent {
     }
 
     if (options.customValidationMessage) {
+      const message = options.customValidationMessage
+
       formSchema = formSchema.messages({
-        any: options.customValidationMessage
+        'any.required': message,
+        'string.empty': message,
+        'string.max': message,
+        'string.min': message,
+        'string.length': message,
+        'string.pattern.base': message
```

Comment:
Can I check we definitely want `customValidationMessage` to override all of these?

For example, a custom message for invalid formatting will also show for min/max length